server/dive_server/serializers/viame.py

```diff
@@ -159,13 +159,70 @@ def _parse_row_for_tracks(row: List[str]) -> Tuple[Feature, Dict, Dict, List]:
     return feature, attributes, track_attributes, confidence_pairs
 
 
-def load_csv_as_tracks(rows: List[str]) -> Dict[str, dict]:
+def create_attributes(
+    metadata_attributes: Dict[str, Attribute],
+    test_vals: Dict[str, int],
+    atr_type: str,
+    key: str,
+    val,
+):
+    valstring = f'{val}'
+    if f'{atr_type}_{key}' not in metadata_attributes:
+        metadata_attributes[f'{atr_type}_{key}'] = {
+            'belongs': atr_type,
+            'datatype': 'text',
+            'name': key,
+            'key': f'{atr_type}_{key}',
+        }
+        test_vals[f'{atr_type}_{key}'] = {}
+        test_vals[f'{atr_type}_{key}'][valstring] = 1
+    elif (
+        f'{atr_type}_{key}' in metadata_attributes and f'{atr_type}_{key}' in test_vals
+    ):
+        if valstring in test_vals[f'{atr_type}_{key}']:
+            test_vals[f'{atr_type}_{key}'][valstring] += 1
+        else:
+            test_vals[f'{atr_type}_{key}'][valstring] = 1
```

Comment:
```suggestion python
    # 'type_name' matches the client convention in AttributeEditor.vue
    atr_key_str = f'{atr_type}_{key}'
    if atr_key_str not in metadata_attributes:
        metadata_attributes[atr_key_str] = {
            'belongs': atr_type,
            'datatype': 'text',
            'name': key,
            'key': atr_key_str,
        }
        test_vals[atr_key_str] = {}
        test_vals[atr_key_str][valstring] = 1
    elif (
        atr_key_str in metadata_attributes and atr_key_str in test_vals
    ):
        if valstring in test_vals[atr_key_str]:
            test_vals[atr_key_str][valstring] += 1
        else:
            test_vals[atr_key_str][valstring] = 1
```

replace `f'{atr_type}_{key}'` with a variable since it's critical that all usages be identical.